client/asset/eth/multirpc.go

```diff
@@ -727,14 +740,19 @@ func createAndCheckProviders(ctx context.Context, walletDir string, endpoints []
 				p.shutdown()
 			}
 		}()
-		if len(providers) != len(unknownEndpoints) {
+		if !defaultProviders && len(providers) != len(unknownEndpoints) {
 			return fmt.Errorf("expected to successfully connect to all of these unfamiliar providers: %s",
 				failedProviders(providers, unknownEndpoints))
 		}
-		if err := checkProvidersCompliance(ctx, providers, compat, dex.Disabled /* logger is for testing only */); err != nil {
+		providers, err = checkProvidersCompliance(ctx, providers, compat, dex.Disabled /* logger is for testing only */, !defaultProviders)
+		if err != nil {
 			return err
 		}
+		if len(providers) == 0 {
+			return fmt.Errorf("no compliant providers found among: %s", unknownEndpoints)
+		}
 	}
```

Comment:
I'd be willing to allow connecting with >= 1 compliant providers even for configured providers. Probably just not for initial configuration or during reconfiguration. But on normal connect, for sure.